shared/teams/new-team/index.js

```diff
@@ -26,6 +26,7 @@ const Contents = ({errorText, name, onNameChange, onSubmit}: Props) => (
           ...stylePadding,
           alignItems: 'center',
           justifyContent: 'center',
+          backgroundColor: globalColors.white,
```

Comment:
did you mean to do this? if it was to fix an earlier bug where the popup was transparent, I think that's fixed now

Reply:
will test that too. That took me, like, a half a day to figure out.

Reply:
nope, still transparent without this on mobile.